content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -191,40 +191,56 @@ configuration parameters from your AppID:
    ```
 
 3. Update the `dist/stacks/ibm/application/oidc-authservice-appid/params.env`
-   and `dist/stacks/ibm/application/oidc-authservice-appid/secret_params.env`
    with values collected in [Prereq](#prerequisites-1) section.
    You will need the following values:
-    * `<oAuthServerUrl>` - fill in the value of oAuthServerUrl
-    * `<clientId>` - fill in the value of clientId
-    * `<secret>` - fill in the value of secret
+    * `<oAuthServerUrl>` - replace `<APP_ID_oauthServerUrl>`
     * `<kubeflow-FQDN>` - fill in the FQDN of Kubeflow, if you don't know yet, just give a dummy one like
       `localhost`. Then change it after you got one. Or get default FQDN of your cluster by this command:
       `ibmcloud ks nlb-dns ls -c <cluster name>` (replace `<cluter name>` with your cluster name)
 
+   Example:
+   ```
+   OIDC_PROVIDER=https://us-south.appid.cloud.ibm.com/oauth/v4/f341ff8b-a088-497a-same-5da4628df7fd
+   REDIRECT_URL=https://my-kubeflow-442dbba0442be6c8c50f31ed96b00601-0000.sjc04.containers.appdomain.cloud/login/oidc
+   OIDC_AUTH_URL=https://us-south.appid.cloud.ibm.com/oauth/v4/f341ff8b-a088-497a-same-5da4628df7fd/authorization
+   ```
+
+4. Update the `dist/stacks/ibm/application/oidc-authservice-appid/secret_params.env`
+   with values collected in [Prereq](#prerequisites-1) section.
+   You will need the following values:
+    * `<clientId>` - replace the `<APP_ID_clientId>`
+    * `<secret>` - replace the `<APP_ID_secret>`
 
-4. You can apply the `kustomize` file in `iks-multi` folder:
+   Example:
+   ```
+   CLIENT_SECRET=NjNhZDA3ODAtM2I3MCSECRETLTkwN2QtNDdhYmU5ZGIyMTBl
+   CLIENT_ID=52b3e496-8888-8888-ABC9-c0da309cdf52
+   ```
+
+5. You can apply the `kustomize` file in `iks-multi` folder:
 
    ```bash
    while ! kustomize build iks-multi | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
    ```
 
-5. If at any point the values change and you have to change them, you can either patch the
+6. If at any point the values change and you have to change them, you can either patch the
    [configmap](#patch-configmap) and [secret](#patch-secret) or change the content in the
    files and apply the kustomize again. You will need to restart authservice with
-   `kubectl rollout restart statefulset authservice -n istio-system` .
+   `kubectl delete pod -l app-authservice -n istio-system` .
```

Comment:
`rollout restart` sometimes can't catch up the new configmap and pod deletion is more reliable.

Reply:
`kubectl rollout restart` is the right way to refresh pod(s) in a statefulset/daemonset/deployment to perfrom rolling updates.

you could restart pods by deleting them, it's not recommended.

Reply:
when rolling out statefulset, it might not able to create a new pod. Is it possible to roll out a new pod instead?

Reply:
I don't see any option to force the  new pod creation.

Reply:
> when rolling out statefulset, it might not able to create a new pod.

need to learn more about it.

Again, I already approved this PR, we can exclude such discussion out of this one